chapter_12/chapter_12.tex

```diff
@@ -1,5 +1,10 @@
 \chapter{Topological Data Analysis}
 \begin{refsection}
 %\input{chapter_12/...}
+\input{chapter_12/0_Intro_TDA}
+\input{chapter_12/1_Metric_Space}
+\input{chapter_12/2_Mapper_algorithm}
+\input{chapter_12/3_Reconstruction}
+\input{chapter_12/4_ Persist_Homology}
```

Comment:
The file name has a space between 4_ and Persist which breaks compilation, please remove it.